xoinvader/background.py

```diff
@@ -93,7 +88,7 @@ def load_chunks(filename, trim_width=None):
                 chunks.append(current_chunk)
                 names.append(name)
                 continue
-            elif not current_chunk:
+            elif current_chunk is None:
```

Comment:
Got it. Write about it in the commit message then. I mean that that is actually `if bool(current_chunk)` and `bool` calls `__len__` method if it's present and so on.

Anyway, checking for `None` should happen as `foo is None`, and this is primarily style issue.